tests/cache/tests.py

```diff
@@ -239,14 +239,14 @@ def custom_key_func(key, key_prefix, version):
 }
 
 
-def caches_setting_for_tests(base=None, **params):
+def caches_setting_for_tests(base=None, exclude=(), **params):
```

Comment:
I almost did that first time around, but didn't since we weren't modifying it.

However someone's bound to copy-pasta it somewhere, so I agree we should do the right thing :-)